custom-commands/general-log/general_log

```diff
@@ -58,4 +59,4 @@ function shutdown() {
 trap shutdown INT
 
 enable
-tail -f /var/lib/mysql/www.log 2> /dev/null
+tail -f /var/lib/mysql/general_log.log 2> /dev/null
```

Comment:
I don't see any reason to redirect stderr to /dev/null - just confuses people if an error should occur. Not redirecting would have made for an understandable error in the bug this is fixing.

Reply:
yes, good idea.